packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -42,6 +54,14 @@ export class EdDSAPCD implements PCD<EdDSAPCDClaim, EdDSAPCDProof> {
   }
 }
 
+let eddsa: Eddsa;
+let poseidon: PoseidonFn;
+
+export async function init(): Promise<void> {
+  eddsa = await buildEddsa();
+  poseidon = await buildPoseidon();
+}
```

Comment:
i recommend adding an `isInitialized` variable to this module and checking whether it's true in `prove` and `verify` and other functions, and throwing an exception if it's not. You're also going to need to invoke `init` in both `passport-client` and `passport-server` for this package. I'm not sure if you added the eddsa pcd package to the appropriate places in the client and server. Whenever we add a new type of pcd, we should also make sure that it works end-to-end in the actual application. One way to do that is to modify `consumer-client` to add a button that adds the new pcd to the passport.